packages/form-schema/src/components/CheckboxesWidget.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
```

Comment:
This file contained a lot of es-lint errors when I copied it into the project. I'm not entirely sure how/what needs fixing here.

Reply:
es errors or ts errors? We haven't been using prop types, just typescript for props s lines 59-82 can go. If they are ts-errors I would put `// ts-ignore` at the top of the file first just to check it's working